vector-app/src/debug/java/im/vector/app/features/debug/jitsi/DebugJitsiActivity.kt

```diff
@@ -31,15 +30,16 @@ class DebugJitsiActivity : VectorBaseActivity<ActivityDebugJitsiBinding>() {
 
     @SuppressLint("SetTextI18n")
     override fun initUiAndData() {
-        val isCrashReportingDisabled = JitsiMeet.isCrashReportingDisabled(this)
-        views.status.text = "Jitsi crash reporting is disabled: $isCrashReportingDisabled"
-
-        views.splash.setOnClickListener {
-            JitsiMeet.showSplashScreen(this)
-        }
-
-        views.dev.setOnClickListener {
-            JitsiMeet.showDevOptions()
-        }
+        // Tchap: Jitsi is only available on withvoip flavor, just comment the following lines to remove the dependency
```

Comment:
I didn't because this a debug activity, I am not sure it is relevant to have it in Tchap, I added a comment to track this, if we want to have it one day, we can move it to the right package. If you prefer to do it now, I can move the class to `withvoip/debug` folder but I'll also have to create a dedicated manifest and a fake activity for the main package, it's a bit painful in my point of view